src/metal.js

```diff
@@ -1,3 +1,21 @@
-// Add Backbone.Metal extensions here
-// 1. Marionette.Class
-// 2. Marionette.Error and Marionette.Deprecate
+// Missing Metal Items
+// 1. Marionette.Class parity
+// 2. Marionette.Deprecate
+
+Marionette.Class = Metal.Class.extend({});
+
+function classify(obj) {
+  return Marionette.Class.extend(
+    _.extend({constructor: obj}, _.omit(obj.prototype, _.keys(Backbone.Events))),
+    _.omit(obj, _.keys(Marionette.Class))
+  );
+}
+
+Marionette.Mixin      = Metal.Mixin;
+Marionette.Events     = Backbone.Events     = new Marionette.Mixin(Backbone.Events);
+Marionette.Model      = Backbone.Model      = classify(Backbone.Model);
+Marionette.Collection = Backbone.Collection = classify(Backbone.Collection);
+Marionette.View       = Backbone.View       = classify(Backbone.View);
+Marionette.Router     = Backbone.Router     = classify(Backbone.Router);
+Marionette.History    = Backbone.History    = classify(Backbone.History);
+Marionette.history    = Backbone.history    = new Marionette.History();
```

Comment:
It would be cool if you could use this to check the order is correct.

```js
if (Backbone.history.handlers.length || Backbone.history.started) {
  throw new Error('You must import Marionette before adding routes or starting history`);
}
```